main.py

```diff
@@ -8,13 +8,48 @@
 # Get the appropriate Slack channel, or default to #cronut-dev
 channel_name = os.environ.get('CHANNEL_NAME','#cronut-dev') 
 
-try:
-    response = client.chat_postMessage(
-        channel=channel_name,
-        text="Hello world!")
-    assert response["message"]["text"] == "Hello world!"
-except SlackApiError as e:
-    # You will get a SlackApiError if "ok" is False
-    assert e.response["ok"] is False
-    assert e.response["error"]  # str like 'invalid_auth', 'channel_not_found'
-    print(f"Got an error: {e.response['error']}")
\ No newline at end of file
+def hello_world():
+    """
+    Test that the API client is working
+    """
+    try:
+        response = client.chat_postMessage(
+            channel=channel_name,
+            text="Hello world!")
+        assert response["message"]["text"] == "Hello world!"
+    except SlackApiError as e:
+        # You will get a SlackApiError if "ok" is False
+        assert e.response["ok"] is False
+        assert e.response["error"]  # str like 'invalid_auth', 'channel_not_found'
+        print(f"Got an error: {e.response['error']}")
+
+def get_channel_id(client, channel_name):
+    channel_id = None
+    try:
+        response = client.conversations_list()
```

Comment:
Curious if this returns private conversations, too. Perhaps if we limit the scoping maybe not?

Reply:
Just pushed an update to limit the pull to public channels.